components/usage/pkg/apiv1/billing.go

```diff
@@ -44,20 +44,19 @@ func (s *BillingService) ReconcileInvoices(ctx context.Context, in *v1.Reconcile
 		return nil, status.Errorf(codes.Internal, "Failed to reconcile invoices.")
 	}
 
-	//TODO (se) make it one query
-	stripeBalances := []db.Balance{}
```

Comment:
Diff shows up weird, but we had 2 loops. First loop filtered out any attribution IDs which did not have a cost center set to stripe, second computed the summary. They've been merged.